### DISCUSSION TITLE
Sort by feature for each Note

### DISCUSSION THREAD

**Discussion by jackthebean** [FossifyOrg/General-Discussion#607] [Ideas]
App version: 1.1.0
Device OS: 10
------------------------------
Thanks for taking this on. At the moment Notes lets you sort a Note's entries but this sorting choice applies to every Note. It would be great if each Note could have its own sorting criteria.

**Comment by Aga-C** [comment]
As I wrote in https://github.com/FossifyOrg/Notes/discussions/79, please create [a new feature request](https://github.com/FossifyOrg/Notes/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for it, not another discussion.
